src/Symfony/Component/Debug/DebugClassLoader.php

```diff
@@ -313,6 +316,17 @@ public function checkAnnotations(\ReflectionClass $refl, $class)
             return $deprecations;
         }
 
+        if (
+            isset($parentAndOwnInterfaces['Serializable']) &&
+            0 === strpos($refl->getNamespaceName(), 'Symfony\\') &&
+            (
+                (!$refl->hasMethod('__serialize') || !($methodRefl = $refl->getMethod('__serialize'))->isPublic() || $methodRefl->isStatic() || !$refl->hasMethod('__unserialize') || !($methodRefl = $refl->getMethod('__unserialize'))->isPublic() || $methodRefl->isStatic()) &&
```

Comment:
@nicolas-grekas Should I add a check on arguments and return types ?

Reply:
nope, not our job IMHO

Reply:
I would suggest to also remove all the checks related to the modifiers of the methods (public/static)